### PR TITLE
[react-typing-effect] Support eraseSpeed prop

### PR DESCRIPTION
**Pull request by rossdakin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/lamyfarai/react-typing-effect/commit/da6d3180fe75507bb04e310220b33f627b7eff90
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

cc @michael-goss @lamyfarai

### CODE REVIEW

**Review by Debananda — APPROVED** [review]
Changes looks good

File: types/react-typing-effect/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-typing-effect 2.0.4
```
Comment (Debananda) [inline types/react-typing-effect/index.d.ts]: @rossdakin : any specific reason we have removed the patch version. I can see the source version is 2.0.4

"react-typing-effect": "^2.0.4",

Comment (rossdakin) [reply types/react-typing-effect/index.d.ts]: @Debananda – I added the `.4` but apparently I shouldn't have, according to [this rule](https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md) which caused the build to fail, so I reverted it to `2.0` as it was.

**Review by Debananda — APPROVED** [review]
Looks good

**Comment by rossdakin** [comment]
Ready to merge

**Pull request merged**